<?xml version="1.0"?>
<launch>
    <node pkg="ndt_fuser" type="ndt_fuser_node" name="fuser_3d" output="screen">
	<param name="points_topic" value="/velodyne_points" />
	<param name="matchLaser" value="false" />
	<param name="size_x_meters" value="150" />
	<param name="size_y_meters" value="150" />
	<param name="size_z_meters" value="5" />
	<param name="resolution" value="0.4" />
	<param name="sensor_range" value="15." />
	
	<!-- Specific sensor offset parameters for the example file -->
	<param name="sensor_pose_x" value="0.0" />
	<param name="sensor_pose_z" value="0.8" />
	<param name="sensor_pose_t" value="0.0" />
	
	<param name="pose_init_x" value="0.0" />
	<param name="pose_init_y" value="0.0" />
	<param name="pose_init_t" value="0.0" />
	
	<param name="map_directory" value="$(find ndt_fuser)/maps/" />	
	<param name="map_name_prefix" value="level9fusion" />
	
	<param name="beHMT" value="false" />
	<param name="match2D" value="true" />
	<param name="useOdometry" value="true" />
	<param name="odometry_topic" value="RosAria/pose" />
	<param name="visualize" value="true" />
	<param name="plotGTTrack" value="true" />
	<param name="initPoseFromGT" value="false" />
	<!--<param name="initPoseFromGT" value="true" />-->
	<param name="gt_topic" value="/vmc_navserver/state" />
    </node>
</launch>
